src/react/components/pages/editorPage/editorSideBar.test.tsx

```diff
@@ -117,8 +117,8 @@ describe("Editor SideBar", () => {
         };
 
         const wrapper = createComponent(props);
-        const list = wrapper.find(List).instance() as List;
-        const recomputeRowHeightsSpy = jest.spyOn(list, "recomputeRowHeights");
+        const grid = wrapper.find(Grid).instance() as Grid;
+        const recomputeRowHeightsSpy = jest.spyOn(grid, "recomputeGridSize");
```

Comment:
nit: to keep your method names consistent, this should be renamed "recomputeGridSizeSpy"